app.js

```diff
@@ -10,6 +10,33 @@
         // logic and calculations
         // state update
         // re-display components
+const volumeBox = document.querySelector('section');
+const [down, up] = volumeBox.querySelectorAll('button');
+const volumeDisplay = volumeBox.querySelector('span');
+
+function updateVolume() {
+    volumeDisplay.textContent = volume;
+}
+
+down.addEventListener('click', () => {
+    if (volume < 1) {
+        return;
+    } 
+    else {
+        volume--;
+        updateVolume();
+    }
+});
```

Comment:
Because you have the return in the `if`, you technically don't need the else:

```js
if (volume < 1) return;
    
volume--;
updateVolume();
```

Using an `if` this way is called a guard clause

Reply:
Oh! That looks easier to read too. I will do that from now. Thank you!